doc/source/WorkingWithScenes/LightLinking/index.md

```diff
@@ -0,0 +1,169 @@
+
+![A Set node in the Node Editor](images/interfaceLightSetNodeEditor.png "A Set node in the Node Editor")
+
+To link one or more lights by a set, simply add it to the set expression.
+
+![A light linking set expression with a set](images/taskLightLinkingSetExpressionSet.png "The light linking set expression with a set")
+
+
+## Example Scenario ##
+
+Let's explore an example scenario where you would need to link lights.
+
+Picture a scene where a giant robot attacks a highly detailed metropolitan city at night. For dramatic effect, let's say we want the robot's eyes to shine with some menacing red highlights. What's more, on the robot's chest is affixed the logo of its evil creator, and we want to make sure the audience can just make it out, regardless of the ambient illumination. With a photorealism mindset, both of these lights would be conceits, as they would have no plausible light source. Even so, with their inclusion, the scene becomes more expressive and evocative.
+
+![A preview of the example scenario in the Viewer](images/demoGaffyAttacksGreyShaded.png "A preview of the example scenario in the Viewer")
+
+To achieve both of these highlights, we could add special lights in front of the robot's eyes and chest. But, by default, this light would also illuminate the rest of the robot, and everything else in the scene, however imperceptibly. In a scene of this scale and complexity, the renderer would waste considerable resources calculating rays and their reflections between the special lights and the nearby buildings, streets, cars, crowds, debris, and other objects.
+
+
+### Light linking solution ###
+
+Let's consider a possible light link setup for this scene, with the goal of achieving our artistic intent while reducing the impact of the special lights on render time and memory.
+
+
+### Lights ###
+
+```eval_rst
+=============================== =============================== ===============================
+Location                        `Default Light` plug value      Special light sets
+=============================== =============================== ===============================
+/lights/sky                     :code:`True`
+/lights/moon                    :code:`True`
+/lights/eyes                    :code:`False`                   "lights:eyes"
+/lights/logo                    :code:`False`                   "lights:logo"
+=============================== =============================== ===============================
+```
+
+
+### Objects ###
+
+```eval_rst
+============================================================== ===============================
+Location                                                       `Linked Lights` expression
+============================================================== ===============================
+/assets/...                                                    –
+/assets/GAFFERBOT/[etc]/L_eyeLens001_REN                       :code:`defaultLights lights:eyes`
```

Comment:
I was sorely tempted to use `...` instead of `[etc]` but the former is a proper token...

Reply:
Yeah good call